platform/admin/core/admin-core-appservice/src/main/java/org/codice/ddf/admin/application/rest/ApplicationUploadEndpoint.java

```diff
@@ -204,7 +202,9 @@ private File createFileFromAttachement(Attachment attachment, Response response)
                 try {
                     File uploadDir = new File(defaultFileLocation);
                     if (!uploadDir.exists()) {
-                        uploadDir.mkdirs();
+                        if(!uploadDir.mkdirs()) {
+                            LOGGER.warn("Unable to make directory");
```

Comment:
after talking with @lessarderic decided to handle this with setting the returned newFile to null which the calling method handles appropriately already.